website/docs/index.html.markdown

```diff
@@ -1,108 +1,88 @@
+in the immediate future, so direct contributions to that resource are discouraged.
+* An issue assigned to `hashibot` indicates a member of the community has taken on
+the issue!
 
-* `project` - (Optional) The ID of the project to apply any resources to.  This
-  can also be specified using any of the following environment variables (listed
-  in order of precedence):
+## Contributing
 
-    * `GOOGLE_PROJECT`
-    * `GOOGLE_CLOUD_PROJECT`
-    * `GCLOUD_PROJECT`
-    * `CLOUDSDK_CORE_PROJECT`
+If you'd like to help extend the Google provider, we gladly accept community
+contributions! Check out the [provider GitHub repo README](https://github.com/terraform-providers/terraform-provider-google)
+for instructions about getting started developing, the [HashiCorp contribution guidelines](https://github.com/hashicorp/terraform/blob/master/.github/CONTRIBUTING.md)
+for a Terraform provider development overview, and the [Google provider contribution guidelines](https://github.com/terraform-providers/terraform-provider-google/blob/master/.github/CONTRIBUTING.md)
+for our provider-specific advice.
 
-* `region` - (Optional) The region to operate under, if not specified by a given resource.
-  This can also be specified using any of the following environment variables (listed in order of
-  precedence):
+## GCP API Versions
 
-    * `GOOGLE_REGION`
-    * `GCLOUD_REGION`
-    * `CLOUDSDK_COMPUTE_REGION`
+The Google provider supports generally available (GA) and Beta GCP features. We
+are focusing on filling out general GA feature coverage and on adding support
+for beta features that customers request. So if you need us to support a feature
+whether GA or beta, please [file a feature request](https://github.com/terraform-providers/terraform-provider-google/issues/new?template=enhancement.md)!
 
-* `zone` - (Optional) The zone to operate under, if not specified by a given resource.
-  This can also be specified using any of the following environment variables (listed in order of
-  precedence):
+If you're interested in using Alpha GCP features, you should still [file a feature request](https://github.com/terraform-providers/terraform-provider-google/issues/new?template=enhancement.md)
+or thumbs up [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
+the existing request if one exists. By filing and reacting to requests, we can
+gauge your interest in yet-to-be-supported GCP features and make sure that we
+prioritise support for them when they enter Beta.
 
-    * `GOOGLE_ZONE`
-    * `GCLOUD_ZONE`
-    * `CLOUDSDK_COMPUTE_ZONE`
-
-
-## Beta Features
-
-Some Google Provider resources contain Beta features; Beta GCP Features have no
+### Beta Features
+Some resources contain Beta features; Beta GCP Features have no
 deprecation policy, and no SLA, but are otherwise considered to be feature-complete
 with only minor outstanding issues after their Alpha period. Beta is when a GCP feature
```

Comment:
grammar nit (I know this is in the unchanged part): this sentence starts in singular and ends in plural, let's make it plural all the way through (beta is when gcp features are publicly announced)